.github/workflows/test.yaml

```diff
@@ -5,9 +5,15 @@ on:
   push:
     paths-ignore:
       - 'docs/**'
+      - '**.md'
+      - '.gitignore'
     branches:
       - main
   pull_request:
+    paths-ignore:
```

Comment:
I _think_ the reason Pa11y was still running on documentation-only PRs was that paths-ignore needs to be scoped to the pull_request event as well